open-craft/tutor-mfe
# feat: backport discussion-mfe to nutmeg

## Description

Backports the discussion forum changes made in Olive to the latest nutmeg release.

The `opencraft-release/nutmeg.2` branch was created from the [v14.0.2](https://github.com/overhangio/tutor-mfe/tree/v14.0.2) tag on `tutor`.

## Testing instructions

The changes have been deployed to https://es1.kgdocluster.opencraft.hosting/. Create a new account and enroll into any course. You should be able to view the new discussion experience.

- [Deployed config](https://gitlab.com/opencraft/devstacks/keith/grove-do/-/blob/main/instances/es1/config.yml)
- [Deployed requirements](https://gitlab.com/opencraft/devstacks/keith/grove-do/-/blob/main/instances/es1/requirements.txt)
